### ISSUE TITLE
Integer Field not accepting 0 as a value

### ISSUE THREAD

**Issue by CreativeWolf** [allcount/allcountjs#100]
Integer Field is not accepting 0 as a value and when it's defined as a required field and given 0 as a value, behaves as if 0 is not a value.

**Comment by CreativeWolf** [comment]
Can this line of code:
```if (field.isRequired && (_.isUndefined(fieldValue) || _.isNull(fieldValue))) {
        return "Required";
}```

be modified as:

```if (field.isRequired && (_.isUndefined(fieldValue) || field.length === 0)) {
        return "Required";
}```

to address this issue?

**Comment by paveltiunov** [comment]
Please send link to this line.

**Comment by CreativeWolf** [comment]
https://github.com/allcount/allcountjs/blob/9565fd684904b94717facddfa9c4a4f575fac48e/services/validation-service.js

**Comment by paveltiunov** [comment]
Fixed in #111
